.circleci/config.yml

```diff
@@ -18,23 +18,20 @@ jobs:
       - run:
           name: install dependencies and package
           command: |
-            python -m venv skimage_venv
-            source skimage_venv/bin/activate
             python -m pip install --upgrade pip
             python -m pip install -r requirements/build.txt
-            python -m pip install -vv --no-build-isolation .
             python -m pip install -r requirements/default.txt
+            python -m pip install -r requirements/optional.txt
             python -m pip install -r requirements/docs.txt
-            export
+            python -m pip install -vv --no-build-isolation .
```

Comment:
The order was important to ensure that one could build without the optional dependencies

Reply:
Good to know! Though I would have expected that kind of test elsewhere and not inside the doc build job. I'll revert it.